packages/core/src/tools/cleaner.js

```diff
@@ -43,66 +43,96 @@ const recurseCleanFuncs = (obj, path) => {
   return obj;
 };
 
+const findNextCurlies = (str) => {
+  const start = str.indexOf('{{');
```

Comment:
Is it possible to do this using regext? something like: 

```
const findNextCurlies = (str) => {
  const match = str.match(/{{.*?}}/);
  if (!match) {
    return {
      start: -1,
      end: -1,
    };
  }

  return {
    start: match.index,
    end: match.index + match[0].length,
  };
};
```
Or are you avoiding that on purpose?

Reply:
I assumed `indexOf` should be faster. The efficiency is probably insignificant. But I'm curious, so I did a little [benchmark](https://jsfiddle.net/s019axu2/). The result: The `indexOf` version is ~6x as fast as the regex version.

Reply:
Ah that's really nice to know! Thanks for the extra due-diligence with the benchmark!